sklearn/linear_model/coordinate_descent.py

```diff
@@ -1965,8 +1965,8 @@ class MultiTaskLassoCV(LinearModelCV, RegressorMixin):
     cv : integer or cross-validation generator, optional
         If an integer is passed, it is the number of fold (default 3).
         Specific cross-validation objects can be passed, see the
-        :mod:`sklearn.cross_validation` module for the list of possible
-        objects.
+        :mod:`sklearn.model_selection.split` module for the list of
```

Comment:
For this alone I referenced the submodule... rest (imports / examples) point to the main `model_selection` module! Is that okay? or should I reference the `model_selection` itself here too?